lib/Doctrine/Validator/Driver.php

```diff
@@ -94,7 +94,7 @@ public function __set($arg, $value)
     public function getArg($arg)
     {
         if ( ! isset($this->args[$arg])) {
-            throw new Doctrine_Validator_Exception('Unknown option ' . $arg);
+            throw new Doctrine_Validator_Exception(array('Unknown option ' . $arg));
```

Comment:
This one is actually marked as `array` with a PHP typehint, so this call would fail: https://github.com/diablomedia/doctrine1/blob/master/lib/Doctrine/Validator/Exception.php#L43